### PR TITLE
[Release] Release vizro 0.1.13

### PR DESCRIPTION
**Pull request by petar-qb**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-core/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+
+## Changed
+
+- Apply new design to `Slider` and `RangeSlider` ([#336](https://github.com/mckinsey/vizro/pull/336))
+- Consolidate gaps between selectors ([#336](https://github.com/mckinsey/vizro/pull/336))
+
+## Fixed
+
+- Fix path to built-in Vizro assets when `requests_pathname_prefix` is not the same as `routes_pathname_prefix` ([#358](https://github.com/mckinsey/vizro/pull/358))
+
```
Comment (petar-qb) [inline vizro-core/CHANGELOG.md:26]: Should I add the AgGrid release to the Highlights changelog section?

**Comment by petar-qb** [comment]
@huong-li-nguyen
Should I rename the PR title to: `[Release] Release of vizro-core 0.1.13`?

File: vizro-core/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,26 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.1.13'></a>
+
+# 0.1.13 — 2024-03-12
+
+### Highlights ✨
+
+- Introduce `AgGrid` as a new `Page` component, allowing the usage of
+  [AG Grid](https://www.ag-grid.com/javascript-data-grid/scrolling-scenarios/) in
```
Comment (maxschulz-COL) [inline vizro-core/CHANGELOG.md:21]: why link to the scrolling scenarios?

**Review by maxschulz-COL — CHANGES_REQUESTED** [review]
Just a qick question, then we can go ahead!

File: vizro-core/CHANGELOG.md (hunk 2 above)
Comment (maxschulz-COL) [inline vizro-core/CHANGELOG.md:21]:
```suggestion
  [AG Grid](https://www.ag-grid.com/) in
```

File: vizro-core/CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,26 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+  for more information. ([#289](https://github.com/mckinsey/vizro/pull/289),[#268](https://github.com/mckinsey/vizro/pull/268),[#324](https://github.com/mckinsey/vizro/pull/324))
+
+## Changed
+
+- Apply new design to `Slider` and `RangeSlider` ([#336](https://github.com/mckinsey/vizro/pull/336))
+- Consolidate gaps between selectors ([#336](https://github.com/mckinsey/vizro/pull/336))
+
+## Fixed
+
+- Fix path to built-in Vizro assets when `requests_pathname_prefix` is not the same as `routes_pathname_prefix` ([#358](https://github.com/mckinsey/vizro/pull/358))
```
Comment (huong-li-nguyen) [inline vizro-core/CHANGELOG.md:32]:
```suggestion
- Apply new design to `Slider` and `RangeSlider`. ([#336](https://github.com/mckinsey/vizro/pull/336))
- Consolidate gaps between selectors. ([#336](https://github.com/mckinsey/vizro/pull/336))

## Fixed

- Fix path to built-in Vizro assets when `requests_pathname_prefix` is not the same as `routes_pathname_prefix`. ([#358](https://github.com/mckinsey/vizro/pull/358))
```

File: vizro-core/CHANGELOG.md (hunk 2 above)
Comment (huong-li-nguyen) [reply vizro-core/CHANGELOG.md:21]: Do we need to link to it at all? We don't do it for any of the other dash components e.g.

Would eventually just shorten to:

Introduce `AgGrid` as a new `Page` component. Visit the [user guide on tables](https://vizro.readthedocs.io/en/stable/pages/user_guides/table/) to learn more. ([#289](https://github.com/mckinsey/vizro/pull/289))

**Comment by huong-li-nguyen** [comment]
> @huong-li-nguyen Should I rename the PR title to: `[Release] Release vizro-core 0.1.13`?

No, it's fine as you have it :) Because the pypi package is just called vizro. When you do the bump PR - I would eventually call it vizro-core

**Review by huong-li-nguyen — APPROVED** [review]

**Pull request merged**